README.md

```diff
@@ -9,26 +9,27 @@ audited, might contain bugs and should not be used in production.
 
 ## Table of contents
 
-- [Introduction](#introduction)
-- [Architecture](#architecture)
-- [Contracts](#contracts)
-  - [Overview](#overview)
-  - [More about static_a_token on L2](#more-about-static_a_token-on-l2)
-  - [Proxies](#proxies)
-  - [Governance](#governance)
-- [How does it work?](#how-does-it-work)
-  - [Bridging aTokens from L1 to L2](#bridging-atokens-from-l1-to-l2)
-    - [Approve bridge tokens](#approve-bridge-tokens)
-    - [Transfer from L1 to L2](#transfer-l1-to-l2)
-    - [Transfer from L2 to L1](#transfer-l2-to-l1)
-  - [Synchronisation of rewards on L1 and L2](#synchronisation-of-rewards-on-l1-and-l2)
-  - [Claiming rewards on L2](#claiming-rewards-on-l2)
-  - [Bridging rewards from L2 to L1](#bridging-rewards-from-l2-to-l1)
-- [Installation](#installation)
-  - [Environment](#environment)
-  - [Build the cairo files](#build-cairo-files)
-  - [Start testnets](#start-testnets)
-  - [Run tests](#run-tests)
+- [Aave Starknet Bridge](#aave-starknet-bridge)
```

Comment:
no need to add this line IMO, this is not a section. it just points to the top of the file. plus, it unnecessarily indents all bullet points